command-expansion-server/.prettierrc

```diff
@@ -1,9 +1,11 @@
 {
-  "tabWidth": 2,
-  "parser": "typescript",
+  "trailingComma": "all",
   "singleQuote": true,
-  "semi": true,
   "arrowParens": "avoid",
+  "semi": true,
   "printWidth": 120,
-  "trailingComma": "all"
+  "tabWidth": 2,
+  "useTabs": true,
```

Comment:
[Why Would You Say Something So Controversial Yet So Brave?](https://i.kym-cdn.com/entries/icons/facebook/000/025/090/tumblr_inline_p1brmcd9Dk1rr08jv_500.jpg)

I am a fan of tabs due to its improved behavior in all environments (someone can set their own indent width), but good luck on this one in general!

Reply:
*gets out the popcorn*

Reply:
'spaces vs tabs' !!!

Reply:
### Poll
❤️ for tabs
🚀 for spaces

Reply:
Not to stifle a hearty discussion, but the majority of Aerie repos use spaces. Let's use spaces. :)

Reply:
I don't really care, but the (I think good) argument for tabs is that you can have people with 20/20 vision working on the same project as someone with shite vision as they can configure indentation in their editor to their liking.

I've used both and really dont care, we're all pressing tab to indent anyways.

Reply:
For someone with one bad eye, I do agree with Dylan, but the idea here is to have settings that the developer shouldn't change unless they really have to. That way we don't have formatting noise in our PR. I can make a poll in aerie slack and just use those results for the setting

Reply:
We got spaces now :)